README.md

```diff
@@ -20,18 +20,12 @@ We created this repo with the following goals in mind
 
 ## Install OAM runtime
 
-#### Clone this repo
-
-```console
-git clone git@github.com:crossplane/oam-kubernetes-runtime.git
-cd ./oam-kubernetes-runtime
-```
-
 #### Install OAM controllers
 
 ```console
 kubectl create namespace oam-system
-helm install core-runtime -n oam-system ./charts/oam-kubernetes-runtime
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm install oam --namespace oam-system crossplane-master/oam-kubernetes-runtime --devel
 ```
 
 ## Verify
```

Comment:
Maybe we can move `clone` here.

---
* Clone this repo

```console
git clone git@github.com:crossplane/oam-kubernetes-runtime.git
cd ./oam-kubernetes-runtime
```
---

Reply:
yes, good idea!